access/email/app.go

```diff
@@ -293,12 +293,21 @@ func (a *App) onDeletedRequest(ctx context.Context, reqID string) error {
 }
 
 // getEmailRecipients converts suggested reviewers to email recipients
-func (a *App) getEmailRecipients(ctx context.Context, suggestedReviewers []string) []string {
+func (a *App) getEmailRecipients(ctx context.Context, roles []string, suggestedReviewers []string) []string {
 	log := logger.Get(ctx)
 	recipients := stringset.NewWithCap(len(suggestedReviewers) + len(a.conf.Delivery.Recipients))
```

Comment:
I'm not sure why this method was using stringset, but it looks like this cap doesn't account for roleToRecipients. Want to replace this stringset with a slice to simplify things?

Reply:
Thank you :bow: 
I think it would work anyway because the `cap` is only a hint, and the underlying data structure (map) of the `stringset` would grow as needed

We still need the `Set` property to hold, otherwise we could send the request multiple times to the same email

I created the slice but ended up needing to implement the Set by myself (or using `stringset` at the very end)

What do you think about just not sending a hint for the expected capacity and keep using the `stringset` to hold the data?

Reply:
Ah you're right about needing the stringset, good catch. I think removing the cap is fine.